apps/antalmanac/src/lib/TutorialHelpers.tsx

```diff
@@ -0,0 +1,239 @@
+                    <br />
+                    Use <KbdCard>←</KbdCard> and <KbdCard>→</KbdCard> to navigate the tour.
+                    <br />
+                    Press <KbdCard>Esc</KbdCard> to exit.
+                    <hr />
+                    You can always review the tour by clicking the button in the bottom right corner.
+                </>
+            ),
+        },
+        searchBar: {
+            selector: '#searchBar',
+            content: 'You can search for your classes here!',
+            action: () => {
+                markTourHasRun();
+                setTab(0);
+            },
+            mutationObservables: ['#searchBar'],
+        },
+        importButton: {
+            selector: '#import-button',
+            content: 'Quickly add your classes from WebReg or Zotcourse!',
+        },
+        calendar: {
+            selector: '.rbc-time-view', // Calendar.
+            content: 'See the classes in your schedule!',
+            action: addSampleClasses,
+        },
+        finalsButton: {
+            selector: '#finals-button, #finals-button-pressed',
+            content: (
+                <>
+                    <b>Click</b> to see your finals
+                </>
+            ),
+            /** If the user presses/pressed the finals button, move to the next step */
+            action: tourActionFactory(
+                () => {
+                    goToNamedStep(TourStepName.showFinals);
+                },
+                // Only move to the next step if the button is toggled from off to on, not vice versa.
+                { selector: '#finals-button', eventType: 'click' }
+            ),
+        },
+        showFinals: {
+            selector: '#calendar-root',
+            content: 'See your finals on the calendar',
+            /** Click the finals button if the user hasn't already */
+            action: () => {
+                // If the button has been clicked, we need to wait for the id to change.
+                setTimeout(() => {
+                    document.getElementById('finals-button')?.click();
+                }, 50);
+            },
+        },
+        addedCoursesTab: {
+            selector: '#added-courses-tab',
+            content: (
+                <>
+                    <b>Select</b> the added courses tab for a list of your courses and details
+                </>
+            ),
+            action: tourActionFactory(() => goToNamedStep(TourStepName.addedCoursePane), {
+                selector: '#added-courses-tab',
+                eventType: 'click',
+            }),
+        },
+        addedCoursePane: {
+            selector: '#course-pane-box',
+            content: (
+                <>
+                    <b>Select</b> the added courses tab for a list of your courses and details
+                </>
+            ),
+            action: () => setTab(1),
+            mutationObservables: ['#course-pane-box'],
+        },
```

Comment:
Because the text here is the same, I'm wondering if we can have `addedCoursePane` highlight/mention some of the features on the added pane.

Reply:
I think it'd be better not to because the page doesn't have particularly important stuff, so we shouldn't go into it in too much detail.